src/factory/index.js

```diff
@@ -101,10 +101,15 @@ const transitionFactory = (...args: Array<any>) => {
           const endVal = getPrimitiveValue(end, index);
           const transitionName = camelCase(transition.transition);
 
+          styles.exited[transitionName] = getStyleString(
+            transitionName,
+            styles.exited[transitionName],
+            transition.getStartStyle(startVal)
+          );
```

Comment:
noice, only thing I'll say is a preference to put this after `styled.exiting` to have these in chronological order

Reply:
technically every component starts from the exited state

sidenote: what is this currentStyle thing for `transforms`?